scripts/bower/publish.sh

```diff
@@ -75,6 +75,37 @@ function prepare {
     replaceJsonProp "bower.json" "angular.*" ".*" "$NEW_VERSION"
     replaceJsonProp "package.json" "version" ".*" "$NEW_VERSION"
     replaceJsonProp "package.json" "angular.*" ".*" "$NEW_VERSION"
+    deleteJsonProp "package.json" "main"
+
+    echo "-- Adding CommonJS index file"
+    echo "require('./$repo');" > index.js
+    echo "" >> index.js
+    if [ $repo == "angular" ]
+    then
+      echo "module.exports = angular;" >> index.js
+    else
+      # convert to module names (angular-animate >> ngAnimate)
+      suffix=`echo $repo | cut -c9-`
+      first=`echo $suffix | cut -c1 | sed -e 'y/abcdefghijklmnopqrstuvwxyz/ABCDEFGHIJKLMNOPQRSTUVWXYZ/'`
+      tail=`echo $suffix | cut -c2-`
+
+      if [ $repo != "angular-mocks" ]
+      then
+        echo "module.exports = 'ng$first$tail';" >> index.js
+      else
+        echo "exports.ngMock = 'ngMock';" >> index.js
+        echo "exports.ngMockE2E = 'ngMockE2E';" >> index.js
+        echo "exports.ngAnimateMock = 'ngAnimateMock';" >> index.js
+      fi
+
+      # add angular as a peer dependency
+      deleteJsonProp "package.json" "peerDependencies"
+      replaceInFile "package.json" "homepage\"\: \"http\:\/\/angularjs\.org\"$" "homepage\"\: \"http\:\/\/angularjs\.org\","
+      sed -i '' -e /^}/d "package.json"
+      # have to use single line form so deleteJsonProp will work
+      echo "  \"peerDependencies\": { \"angular\": \"$NEW_VERSION\" }" >> package.json
```

Comment:
Yea, I mention their problems and uncertain future, but @caitp pointed out that since these would be locked to a specific version we shouldn't have any issues. I'm happy to remove it, but that means devs will need to explicitly install angular itself, along with the other modules they need. (Which seems fine, just pointing out.)